src/lib/components/TokenSelect/index.tsx

```diff
@@ -48,9 +52,11 @@ export function TokenSelectDialog({ onSelect }: { onSelect: (token: Currency) =>
         </Row>
         {Boolean(baseTokens.length) && (
           <Row pad={0.75} gap={0.25} justify="flex-start" flex>
-            {baseTokens.map((token) => (
-              <TokenBase value={token} onClick={onSelect} key={token.wrapped.address} />
-            ))}
+            {baseTokens
+              .filter((token) => token !== value)
+              .map((token) => (
+                <TokenBase value={token} onClick={onSelect} key={token.wrapped.address} />
```

Comment:
isn't there a possibility of conflict between native and wrapped native?

can't you use `currencyId(token)` instead as the key?

Reply:
yes and yes